packages/vscode-extension/src/commands/ImportFiles.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+import * as path from 'path';
+import * as vscode from 'vscode';
```

Comment:
I think we can remove this since the line below imports specific items from vscode.